src/Field/Base/PartsField.php

```diff
@@ -150,6 +150,33 @@ final public function addInputContainerAttributes(array $attributes): static
         return $new;
     }
 
+    /**
+     * Replace input container tag CSS classes with a new set of classes.
+     *
+     * @param string|null ...$class One or many CSS classes.
+     */
+    final public function inputContainerClass(?string ...$class): static
+    {
+        $new = clone $this;
+        $new->inputContainerAttributes['class'] = array_filter($class, static fn ($c) => $c !== null);
```

Comment:
Why do we need null check here?

Reply:
For reset class. For example:

```php
// <field class="red blue">
$field = $field->inputContainerClass('red', 'blue');
...
// <field>
$field = $field->inputContainerClass(null);
```

If not filter null values, then result will be `<field class>` instead of `<field>`.